config/application.rb

```diff
@@ -7,6 +7,9 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+# Load dotenv only in development environment
+Dotenv::Railtie.load if ['development'].include? ENV['RAILS_ENV']
```

Comment:
Do we need Rails to read `.env` on development readily? Doesn't seem like it's being used with regards to this PR.

Or is it just a convenience commit, so that we can use it next time?

Reply:
Yeah. When testing the codaveri feature, you need to add the api key in the env file. Currently, you cant read the .env file without that package

Reply:
Is there a way to mock the contact with Codaveri on development? Or does this API key always have to be a live key?

Reply:
They give us a separate api key to be used for dev only